### PR TITLE
Updating DATS.json for corrected "identifier" and "identifierSource" fields.

### PR DESCRIPTION
**Pull request by emmetaobrien**
This PR corrects `DATS.json` in 36 datasets as listed in https://github.com/CONP-PCNO/conp-dataset/issues/712 to adjust the usage of `identifier` and `identifierSource` fields to correctly store URLs in the former rather than the latter.  Three datasets remain uncorrected, `PERFORM` because it is crawled from Zenodo, and two `mousebytes` datasets as described in https://github.com/CONP-PCNO/conp-dataset/issues/712.

### CODE REVIEW

**Comment by emmetaobrien** [comment]
Update: replaced the two datasets under `mousebytes` with links to forks of those to `conpdatasets` and corrected the identifiers for taxonomic data in their `DATS.json` files

File: .gitmodules [hunk 1]
```diff
@@ -181,13 +181,13 @@
...
 	url = https://github.com/conpdatasets/mousebytesdataset.git
+
 [submodule "projects/mousebytes/MB-Cholinergic-deficient-mouse-model"]
 	path = projects/mousebytes/MB-Cholinergic-deficient-mouse-model
-	url = https://github.com/Rodent-Cognition-Core/MB-Cholinergic-deficient-mouse-model.git
+	url = https://github.com/conpdatasets/MB-Cholinergic-deficient-mouse-model
 [submodule "projects/mousebytes/MB-Co-transmission-of-cholinergic-interneurons"]
 	path = projects/mousebytes/MB-Co-transmission-of-cholinergic-interneurons
-	url = https://github.com/Rodent-Cognition-Core/MB-Co-transmission-of-cholinergic-interneurons.git
+	url = https://github.com/conpdatasets/MB-Co-transmission-of-cholinergic-interneurons
```
Comment (cmadjar) [inline .gitmodules:193]:
```suggestion
	url = https://github.com/conpdatasets/MB-Co-transmission-of-cholinergic-interneurons.git
```

File: .gitmodules [hunk 2]
```diff
@@ -181,13 +181,13 @@
...
 	url = https://github.com/conpdatasets/BigBrain_Raw_Data.git
-
 [submodule "projects/mousebytesdataset"]
 	path = projects/mousebytesdataset
 	url = https://github.com/conpdatasets/mousebytesdataset.git
+
 [submodule "projects/mousebytes/MB-Cholinergic-deficient-mouse-model"]
 	path = projects/mousebytes/MB-Cholinergic-deficient-mouse-model
-	url = https://github.com/Rodent-Cognition-Core/MB-Cholinergic-deficient-mouse-model.git
+	url = https://github.com/conpdatasets/MB-Cholinergic-deficient-mouse-model
```
Comment (cmadjar) [inline .gitmodules:190]:
```suggestion
	url = https://github.com/conpdatasets/MB-Cholinergic-deficient-mouse-model.git
```

**Review by cmadjar — CHANGES_REQUESTED** [review]
@emmetaobrien I think you need to merge conp/master again so that the updates done to the validator by the PR that was just merged can be taken into account.

Also see my suggestions below for the .gitmodules file

After that, I feel your PR will be good to go

**Review by cmadjar — APPROVED** [review]

**Pull request merged**